tests/Resources/Page/MarketplaceConsole/MarketplaceConsole.robot

```diff
@@ -0,0 +1,66 @@
+*** Settings ***
+Library         Collections
+
+*** Keywords ***
+Is SSO Login Page Visible
+  Wait Until Element is Visible            //a[contains(@aria-label,"Log in")]
+  Click Element     //a[contains(@aria-label,"Log in")]
+  sleep   5
+  ${login_prompt_visible} =  Run Keyword and Return Status  Page Should Contain    Enter your email to log in
+  [Return]  ${login_prompt_visible}
+
+Get the OC commands For Cluster registration
+    Launch Cluster Tab from Marketplace
+    sleep    5
+    Wait until Element is Visible          //span[contains(text(),'Generate Secret')]        timeout=20
+    Click Button    Generate Secret
+    Wait Until Keyword Succeeds    30  1   Element Should Be Visible     //button[contains(@class, 'disabled')]/span[contains(text(),'Generate Secret')]
+    sleep  5
+    ${elem} =   Get WebElements      xpath://div[@role='textbox']//pre
+    &{oc_command_dict}   	Create Dictionary
+    @{itemname}   Create List     Create namespace         Red Hat Marketplace Subscription          Red Hat Marketplace Kubernetes Secret    Red Hat Marketplace global pull secret
+    FOR  ${idx}  ${ext_link}  IN ENUMERATE  @{elem}   start=0
+
+        Set To Dictionary       ${oc_command_dict}      ${itemname[${idx}]}    ${ext_link.text}
+
+    END
+    [Return]    ${oc_command_dict}
+
+#Launch Marketplace
+#   Open Browser  ${MARKETPLACE_TEST.URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+
+Wait For Marketplace Page To Load
+   Wait Until Page Contains Element    xpath://a[contains(text(), 'Workspace')]   timeout=15
+   Page Should Contain      Red Hat Marketplace
+   Sleep    3
```

Comment:
Why the sleep at the end?

Reply:
I have seen that pages header gets loaded but not whole content so i added sleep...may be further down the line i will add code to check the other element instead of sleep

Reply:
I have removed sleep from the code